Tophat/Views/LaunchFromLocationMenuItem.swift

```diff
@@ -68,6 +68,8 @@ private extension ArtifactFileFormat {
 		switch self {
 			case .zip:
 				return .zip
+			case .tarGzip:
+				return .init(filenameExtension: "tar.gz")
```

Comment:
Any way we can avoid hardcoding this extension here?  Maybe if we have a more generic way of getting compound path extensions we could use that?